pkg/providers/tinkerbell/reconciler/reconciler.go

```diff
@@ -161,7 +186,8 @@ func (r *Reconciler) ReconcileCNI(ctx context.Context, log logr.Logger, clusterS
 	return r.cniReconciler.Reconcile(ctx, log, client, clusterSpec)
 }
 
-func (r *Reconciler) validateTinkerbellIPMatch(ctx context.Context, clusterSpec *c.Spec) error {
+func (r *Reconciler) validateTinkerbellIPMatch(ctx context.Context, tinkerbellScope *Scope) error {
```

Comment:
why is this change needed?

Reply:
You're right, this doesn't need to be changed since it's not a phase. Changing it back